README.md

```diff
@@ -32,11 +31,11 @@ changes like zero-downtime rolling updates with load balancers easy. More inform
 ## Use Ansible
 
 You can install a released version of Ansible with `pip` or a package manager. See our
-[installation guide](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) for details on installing Ansible
+[Installation guide](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) for details on installing Ansible
```

Comment:
```suggestion
[installation guide](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) for details on installing Ansible
```

Reply:
@gotmax23 - why are you suggesting this goes back to lowercase? The Installation Guide is actually uppercase for both...

Reply:
I don't think `installation guide` is a proper noun here, but yes, having only the first word capitalized is not correct either way.

Reply:
It's the title of a guide and we capitalize our guide titles, so yes, both should be capitalized.

Reply:
oh mea culpa - We don't actually specify this in [our style guide](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#header-notation). In fact, the examples use lowercase on the guide, so @Akasurde 's original edit is correct.

...and we should consider adding this to the style guide AND updating all guide names to match. Cuz currently is is Installation Guide in the left-hand navigation.

Reply:
Ah, I see. I thought the guide was called `Installing Ansible`, but I guess that's a subsection of `Installation Guide`. I edited my suggestion accordingly. Thanks.

Reply:
@samccann, it looks like our responses got misordered. You're saying that it should be all lower case?

Reply:
So the guide itself is currently Installation Guide (all initial caps).  Our style guide doesn't actually spell this out specifically, but our examples use sentence case (aka only first word is capitalized).  So we can discuss tomorrow in the DaWGs meeting.

For this PR  - I think @gotmax23 updated edit is correct (it matches what is currently in the docs).